homeassistant/components/landisgyr_heat_meter/const.py

```diff
@@ -0,0 +1,181 @@
+"""Constants for the Landis+Gyr Heat Meter integration."""
+
+from homeassistant.components.sensor import (
+    SensorDeviceClass,
+    SensorEntityDescription,
+    SensorStateClass,
+)
+from homeassistant.const import TEMP_CELSIUS
+from homeassistant.helpers.entity import EntityCategory
+
+DOMAIN = "landisgyr_heat_meter"
+
+HEAT_METER_SENSOR_TYPES = (
+    SensorEntityDescription(
+        key="heat_usage_gj",
+        icon="mdi:fire",
+        name="Heat usage GJ",
+        native_unit_of_measurement="GJ",
+        state_class=SensorStateClass.TOTAL_INCREASING,
+    ),
+    SensorEntityDescription(
+        key="volume_usage_m3",
+        icon="mdi:fire",
+        name="Volume usage m3",
+        native_unit_of_measurement="m3",
+        state_class=SensorStateClass.TOTAL_INCREASING,
+    ),
+    SensorEntityDescription(
+        key="heat_previous_year_gj",
+        icon="mdi:fire",
+        name="Heat usage previous year GJ",
+        native_unit_of_measurement="GJ",
+        entity_category=EntityCategory.DIAGNOSTIC,
```

Comment:
I've considered this, but the [documentation ](https://developers.home-assistant.io/docs/core/entity/sensor/#available-state-classes) states: "_The state represents a measurement in present time, not a historical aggregation such as statistics or a prediction of the future._"
Since this is the heat usage in the previous year, I concluded this should not be a measurement.

Reply:
You're right, sorry about that